packages/halo-theme/src/custom-elements/ef-text-field.less

```diff
@@ -27,6 +27,13 @@
         color: @scheme-color-primary;
       }
     }
+
+    [part~='button-clear']:hover {
+      color: @input-focused-border-color;
+    }
+    [part~='button']:active {
```

Comment:
`:active` state of the clear button is invisible.

Reply:
Fixed.